cli/pkg/artifact_builder/artifact_builder.go

```diff
@@ -269,14 +269,12 @@ func (ab ArtifactBuilder) RegistryLogin(ctx context.Context) error {
 func getStacksECRSFromTFE(ctx context.Context, tfeWorkspace workspace_repo.Workspace, stackName string) (map[string]*config.RegistryConfig, error) {
 	outs, err := tfeWorkspace.GetOutputs(ctx)
 	if err != nil {
-		log.Debugf("unable to get state outputs from stack workspace %s", stackName)
-		return nil, nil
+		return nil, errors.Errorf("unable to get state outputs from stack workspace %s, cannot determine which ECR repo to push to", stackName)
```

Comment:
hmm. this is actually by design to not throw errrors because its behind a feature flag still. if it throws an error, it will break for others that don't have it turned on. Maybe we can make it an error message instead of returning an error

Reply:
I think I have an idea on how to use the feature flags a little better here